atox/build.gradle.kts

```diff
@@ -19,10 +19,10 @@ android {
         multiDexEnabled = true
     }
     buildTypes {
-        getByName("debug") {
```

Comment:
This doesn't belong in this PR and should either be reverted or at least moved to a separate commit. :x

Reply:
Yeah right